flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/SequentialChannelStateReaderImplTest.java

```diff
@@ -65,49 +68,48 @@
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
 import static java.util.stream.IntStream.range;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** {@link SequentialChannelStateReaderImpl} Test. */
-@RunWith(Parameterized.class)
+@ExtendWith(ParameterizedTestExtension.class)
 public class SequentialChannelStateReaderImplTest {
 
-    @Parameterized.Parameters(
-            name =
-                    "{0}: stateParLevel={1}, statePartsPerChannel={2}, stateBytesPerPart={3},  parLevel={4}, bufferSize={5}")
-    public static Object[][] parameters() {
-        return new Object[][] {
-            {"NoStateAndNoChannels", 0, 0, 0, 0, 0},
-            {"NoState", 0, 10, 10, 10, 10},
-            {"ReadPermutedStateWithEqualBuffer", 10, 10, 10, 10, 10},
-            {"ReadPermutedStateWithReducedBuffer", 10, 10, 10, 20, 10},
-            {"ReadPermutedStateWithIncreasedBuffer", 10, 10, 10, 10, 20},
-        };
+    @Parameters
+    public static List<Object[]> parameters() {
+        ArrayList<Object[]> params = new ArrayList<>();
+        params.add(new Object[] {"NoStateAndNoChannels", 0, 0, 0, 0, 0});
+        params.add(new Object[] {"NoState", 0, 10, 10, 10, 10});
+        params.add(new Object[] {"ReadPermutedStateWithEqualBuffer", 10, 10, 10, 10, 10});
+        params.add(new Object[] {"ReadPermutedStateWithReducedBuffer", 10, 10, 10, 20, 10});
+        params.add(new Object[] {"ReadPermutedStateWithIncreasedBuffer", 10, 10, 10, 10, 20});
+        return params;
```

Comment:
```suggestion
        return Arrays.asList(
                new Object[] {"NoStateAndNoChannels", 0, 0, 0, 0, 0},
                new Object[] {"NoState", 0, 10, 10, 10, 10},
                new Object[] {"ReadPermutedStateWithEqualBuffer", 10, 10, 10, 10, 10},
                new Object[] {"ReadPermutedStateWithReducedBuffer", 10, 10, 10, 20, 10},
                new Object[] {"ReadPermutedStateWithIncreasedBuffer", 10, 10, 10, 10, 20});
```
nit: you don't have to change this if you don't agree.